src/features/metrics/instrument/workers-helper.js

```diff
@@ -26,13 +28,15 @@ function resetSupportability () {
 
 /**
  * Injects code to report Web Workers supportability and usage as metrics, replacing the native API in global scope as a side effect.
- * @param {func} report - handles reporting of supportability metrics to NR1
+ * @param {ContextualEE} ee - the event emitter instance to send metrics through
  * @returns void
  */
-function insertSupportMetrics (report) {
+export function insertSupportMetrics (ee) {
   // Of the 3, the normal worker is the most widely supported, so we can be sure metric was already inserted w/o checking other 2.
   if (origWorker) return
 
+  const report = tag => handle(SUPPORTABILITY_METRIC_CHANNEL, [tag], undefined, FEATURE_NAMES.metrics, ee)
```

Comment:
Nbd, but we can avoid further tying standalone modules like this (with a generic `report` function, similar to CWV lib) to the messy EE by leaving this file as is and passing in:
```suggestion
  insertSupportMetrics(tag => handle(SUPPORTABILITY_METRIC_CHANNEL, [tag], undefined, this.featureName, this.ee))
```
in the instrument/index.js instead.